parm/atm/obs/config/amsua_n19.yaml

```diff
@@ -0,0 +1,323 @@
+obs space:
+  name: amsua_n19
+  obsdatain:
+    obsfile: $(OBS_DIR)/$(OBS_PREFIX).amsua_n19.$(OBS_DATE).nc4
+  obsdataout:
+    obsfile: $(DIAG_DIR)/diag_amsua_n19_$(OBS_DATE).nc4
+  simulated variables: [brightness_temperature]
+  channels: &amsua_n19_channels 1-15
+obs operator:
+  name: CRTM
+  Absorbers: [H2O,O3,CO2]
+  Clouds: [Water, Ice]
+  Cloud_Fraction: 1.0
+  obs options:
+    Sensor_ID: amsua_n19
+    EndianType: little_endian
+    CoefficientPath: $(CRTM_COEFF_DIR)
+obs bias:
+  input file: $(BIAS_DIR)/$(BIAS_PREFIX).amsua_n19.satbias.$(BIAS_DATE).nc4
```

Comment:
We should add a line for the updated bias correction file generated by fv3jedi_var.x.   The output bias correction file would have a date of OBS_DATE.

Reply:
So does that require an `output file` section here? I'm not very familiar with how that works.